configuration/postgres/postgres.go

```diff
@@ -288,18 +320,18 @@ func parseMetadata(cmetadata configuration.Metadata) (metadata, error) {
 	}
 
 	if m.ConnectionString == "" {
-		return m, fmt.Errorf(ErrorMissingConnectionString)
+		return m, fmt.Errorf("missing postgreSQL connection string")
 	}
 
 	if m.ConfigTable != "" {
-		if !allowedChars.MatchString(m.ConfigTable) {
-			return m, fmt.Errorf("invalid table name : '%v'. non-alphanumerics are not supported", m.ConfigTable)
+		if !allowedTableNameChars.MatchString(m.ConfigTable) {
```

Comment:
(Not going to block the PR on this) FYI you can use a single regex to validate the charset and length: https://regex101.com/r/hNxNfd/1